### ISSUE TITLE
Feature request: let user decide where to start

### ISSUE THREAD

**Issue by kuyper** [turtl/js#208]
I do use boards a lot to keep my notes organised. It would be great if I could make turtl start with my board overview instead of "all notes".

**Comment by JulienPalard** [comment]
Also, setting a fake-board like "Notes that are not in any boards" as a start page may be nice for those like us using boards extensively.

**Comment by mat-m** [comment]
"Standalone Notes" for notes without board will be great. It makes more sense than All Notes, where all your "sorted" notes in boards still appears.
Moreover, in the current view, we cannot read the board it belongs to (if any) until you are in edit mode.
Having the "Standalone Notes" will ease their mamagement.

About the where to start: Being able to start in one of the globzl view or inside a specific board would be awesome.
Basically, if we are allowed to start in a specific board, and with feature #207, we are almost done.
